FBTweak/FBTweakShakeWindow.m

```diff
@@ -86,6 +79,36 @@ - (void)_presentTweaks
   }
 }
 
+- (FBTweakViewController * _Nullable)findTweakViewController {
+  UIViewController *visibleViewController = self.rootViewController;
+  while (visibleViewController.presentedViewController != nil) {
+    visibleViewController = visibleViewController.presentedViewController;
+  }
+
+  if (![visibleViewController isKindOfClass:[FBTweakViewController class]]) {
+    return nil;
+  }
+
+  return (FBTweakViewController *)visibleViewController;
+}
+
+- (void)dismissTweaksViewController {
+  FBTweakViewController * _Nullable viewController = [self findTweakViewController];
+  if (viewController) {
+    [self dismissTweaksViewController:viewController];
+  }
+}
+
+- (void)dismissTweaksViewController:(FBTweakViewController *)tweakViewController {
```

Comment:
Can be class method.

Reply:
Not really, eventually it uses `self.rootViewController`.
Also imaging the usage
```objc
  auto _Nullable window = [UIApplication sharedApplication].keyWindow;
  if ([window isKindOfClass:[FBTweakShakeWindow class]]) {
    [(FBTweakShakeWindow *)window dismissTweaksViewController];
  }
```

Changing to `[FBTweakShakeWindow dismissTweaksViewController]` is weird.

Reply:
I think we aren't speaking about the same method.

I'm speaking about
```objc
- (void)dismissTweaksViewController:(FBTweakViewController *)tweakViewController {
  [tweakViewController.view endEditing:YES];
  [tweakViewController dismissViewControllerAnimated:YES completion:NULL];
}
```
which can be static. But it's minor, so it's optional.

Reply:
My bad. I thought it was the public method.
Since it's private, and it doesn't hurt functional code, I'm opting to keep this as is.